### PR TITLE
merge the operator and the admin console

### PR DESCRIPTION
**Pull request by sgalsaleh**

### CODE REVIEW

File: pkg/kotsadm/main.go [hunk 1]
```diff
@@ -292,6 +283,44 @@ func removeUnusedKotsadmComponents(deployOptions types.DeployOptions, clientset
 		}
 	}
 
+	// if there are kotsadm-operator objects, remove (pre 1.50.0)
+	_, err = clientset.AppsV1().Deployments(deployOptions.Namespace).Get(context.TODO(), "kotsadm-operator", metav1.GetOptions{})
+	if err == nil {
+		if err := clientset.AppsV1().Deployments(deployOptions.Namespace).Delete(context.TODO(), "kotsadm-operator", metav1.DeleteOptions{}); err != nil {
+			return errors.Wrap(err, "failed to delete kotsadm-operator deployment")
```
Comment (divolgin) [inline pkg/kotsadm/main.go:290]: How about not erroring out on any of these

File: pkg/operator/client/client.go [hunk 2]
```diff
@@ -0,0 +1,463 @@
...
+	appID := args.AppID
+	sequence := args.Sequence
+	informerStrings := args.Informers
+
+	var informers []appstatetypes.StatusInformer
+	for _, str := range informerStrings {
+		informer, err := str.Parse()
+		if err != nil {
+			log.Printf("failed to parse informer %s: %s", str, err.Error())
+			continue // don't stop
```
Comment (divolgin) [inline pkg/operator/client/client.go:423]: this is the best

File: pkg/kotsadm/operator.go [hunk 3]
```diff
@@ -1,274 +0,0 @@
...
-	if err != nil {
-		return errors.Wrap(err, "failed to decode application metadata")
-	}
-
-	if gvk.Group != "kots.io" || gvk.Version != "v1beta1" || gvk.Kind != "Application" {
-		return errors.New("application metadata contained unepxected gvk")
-	}
-
-	application := obj.(*kotsv1beta1.Application)
-	for _, additionalNamespace := range application.Spec.AdditionalNamespaces {
```
Comment (divolgin) [inline pkg/kotsadm/operator.go:122]: I don't see where we crate kotsadm RBAC in all application namespaces

**Review by divolgin — APPROVED** [review]
We need to check that kotsadm

File: pkg/kotsadm/operator.go (hunk 3 above)
Comment (sgalsaleh) [reply pkg/kotsadm/operator.go:122]: ah nice catch, i bet that's why the multi-namespaces test is failing

**Review by divolgin — APPROVED** [review]

**Review by divolgin — APPROVED** [review]

**Review by divolgin — APPROVED** [review]

**Pull request merged**